napari/utils/_testsupport.py

```diff
@@ -149,6 +149,21 @@ def make_napari_viewer(
     from napari._qt.qt_viewer import QtViewer
     from napari.settings import get_settings
 
+    gc.collect()
+
+    _do_not_inline_below = len(QtViewer._instances)
+    # # do not inline to avoid pytest trying to compute repr of expression.
+    # # it fails if C++ object gone but not Python object.
+    if request.config.getoption(_SAVE_GRAPH_OPNAME):
+        fail_obj_graph(QtViewer)
+    QtViewer._instances.clear()
+    assert _do_not_inline_below == 0, (
```

Comment:
Should this string get a translation? Can come later - but i just want to flag

Reply:
This string should be only visible if test crash.  If I good understand translation should cover only strings which will appear when using napari as application?

Reply:
that's a fair point, we don't need to worry about it now